requirements/base.txt

```diff
@@ -1,2 +1,2 @@
 boto3>=1.19.5,==1.*
-jsonschema~=3.2
+jsonschema~=4.6
```

Comment:
I have been discussing with @kddejong. Can we accept greater than 3.2 but less than 5.0 instead? This should allow customer to use 4.x but not force anymore to upgrade. We can later upgrade completely but will help CFN Lint as this rolls out.

Reply:
I tend to like the flexibility.

But the issue is the test results from using 3.2 and from using 4.x are different. There are two test cases which should be caught as invalid but were not when using 3.2. I can look further why 3.2 would not catch these two test cases and see if it's a configuration/schema issue or it's a limitation in 3.2.

Reply:
For test runs, maybe we could check what version is installed and very the right text was returned?

The tests you updated are for a project we aren't actively doing anymore and therefore the json schemas are just overhead. I wonder if the right thing is just to remove these now. Once the project comes back up, we can also revert the removal and handle things that need to be updated?